feg/gateway/services/n7_n40_proxy/n7/config.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package n7
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/golang/glog"
+
+	mcfgprotos "magma/feg/cloud/go/protos/mconfig"
+	"magma/feg/gateway/utils"
+	"magma/gateway/mconfig"
+)
+
+const (
+	N7N40ProxyServiceName = "n7_n40_proxy"
+
+	PcfApiRoot              = "PCF_API_ROOT"
+	PcfTokenUrl             = "PCF_TOKEN_URL"
+	PcfClientId             = "PCF_CLIENT_ID"
+	PcfClientSecret         = "PCF_CLIENT_SECRET"
+	N7ConsumerLocalAddr     = "N7_CONSUMER_LOCAL_ADDR"
+	N7ConsumerNotifyApiRoot = "N&_CONSUMER_NOTIFY_API_ROOT"
+
+	DefaultPcfApiRoot        = "https://localhost"
+	DefaultPcfTokenUrl       = "https://localhost/token"
+	DefaultClientId          = "magma_client_id"
+	DefaultClientSecret      = "magma_client_secret"
+	DefaultN7ConsumerAddr    = "localhost:0"
```

Comment:
This line is fine, but keep in mind that your client configuration will require a specific port. So you will always have to define a port. The reason is, the server PCF needs to be configured with an address and port. If you use 0, your port will be random and will change in every restart. So your PCF will never have the right port configured in their end


For testing we use 0 because we don;'t want to collide with any port used by the machine you run the test.